test/functional/eval/system_spec.lua

```diff
@@ -174,6 +174,19 @@ describe('system()', function()
       end)
     end
 
+    it('works with powershell w/ UTF-8 text (#13713)', function()
+      if not helpers.has_powershell() then
+        pending("not tested; powershell was not found", function() end)
+        return
+      end
+      -- Should work with recommended config used in helper
+      helpers.set_shell_powershell()
+      eq('ああ\n', eval([[system('Write-Output "ああ"')]]))
+      -- Powershell expected to garble text without UTF8 console config
+      command([[let &shellcmdflag = '-NoLogo -NoProfile -ExecutionPolicy RemoteSigned -Command ']])
+      eq('??\n', eval([[system('Write-Output "ああ"')]]))
```

Comment:
Doesn't this depend on default input/output encoding? pwsh uses utf8 on linux.

Reply:
Right, it does, should've tested in Linux env before pushing. Test now checks for different references for win and linux, PTAL